.zuul.d/config.yaml

```diff
@@ -35,6 +35,8 @@
       kaas: false
       do_provision: false
       do_cleanup: true
+      sonobuoy_tar_gz_url: >
+        https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.57.3/sonobuoy_0.57.3_linux_amd64.tar.gz
```

Comment:
I do it this way:

Definition of sonobuoy_version : https://github.com/regiocloud/e2e-tests/blob/main/playbooks/gardener/pre.yml#L15

wget and removal: https://github.com/regiocloud/e2e-tests/blob/main/playbooks/gardener/pre.yml#L58-L60

But I think it makes sense to have a sonobuoy_url like https://github.com/vmware-tanzu/sonobuoy/releases/download/v{{ sonobuoy_version }}/sonobuoy_{{ sonobuoy_version }}_linux_amd64.tar.gz here. This way it's easier to use a local mirror.